loopchain/baseservice/node_subscriber.py

```diff
@@ -53,11 +51,12 @@ class UnregisteredException(Exception):
     pass
 
 
-def convert_response_to_dict(response: bytes) -> dict:
-    response_dict: dict = json.loads(response)
-    response_dict = _check_error_in_response(response_dict)
+def validate_response(response: str) -> str:
+    res: dict = json.loads(response)
+    res = _check_error_in_response(res)
```

Comment:
I agree on your opinion.
I hope this will be treated in better way, in another PR soon. 

I think the PR can touch below:
 In latest version of `jsonrpcserver`, the library validates JSON-RPC schema in silent way. But some points in `loopchain` validates JSON-RPC schema again after the library does.